app/helpers/hosts_extensions_helper.rb

```diff
@@ -0,0 +1,62 @@
+module HostsExtensionsHelper
+  def hosts_multiple_actions
+    return unless can_schedule_jobs?
+
+    [{ action: [_('Schedule Remote Job'), new_job_invocation_path, false], priority: 1000 }]
+  end
+
+  def host_overview_buttons(host)
+    [
+      { button: link_to_if_authorized(_("Jobs"), hash_for_job_invocations_path(search: "host=#{host.name}"), title: _("Job invocations"), class: 'btn btn-default'), priority: 200 },
+    ]
+  end
+
+  def host_title_actions(*args)
```

Comment:
This can also use `describe_host` (method `title_actions_provider`)

Reply:
I tried but it didn't work, at least not the way how it is described in dev docs.
This way was the only one for me that worked.

Reply:
interesting... ok, I did not try myself so ok :+1: this should work pretty safe hopefully (I believe that plugin helpers will always be included after core helpers)